types/node/index.d.ts

```diff
@@ -7917,6 +7924,7 @@ declare module "http2" {
         peerMaxConcurrentStreams?: number;
         selectPadding?: (frameLen: number, maxFrameLen: number) => number;
         settings?: Settings;
+        createConnection?: (option: SessionOptions) => stream.Duplex;
```

Comment:
Make this `createConnection?(...): stream.Duplex`

Reply:
Not quite sure I understand, the function takes a callback that gets the SessionOptions as args and returns a stream.Duplex ?